welcome.md

```diff
@@ -0,0 +1,110 @@
+
+### Command line editors
+Command line editors are editors you can use from the terminal, locally or on the server. The above mentioned `vim` or `emacs` usually come pre-installed on most \*nix Operating Systems. If you really don't like using that on a server you can also use `nano`.
+
+## Package Manager
+Both platforms make it easy to maintain and release software without the need to download seperate packages, but using a package manager.
+
+JavaScript's defacto package manager is excellent and called [npm](https://npmjs.com). It is the largest collection language-specific software in the world and is packaged with [node](https://nodejs.org).
+
+Python uses [pip](https://pip.pypa.io/en/stable/installing/). You can use it in conjunction with virtualenvwrapper, which is a tool to seperate your packages important to your project from the rest of your system.
+
+# Learning JavaScript
+JavaScript has a superpower that no other language can compete with: it runs in every browser. Which means you can start learning right away without even downloading extra stuff.
+The other superpower JavaScript has is that it is slowly taking over the world. Almost anything you can think of has also been written for, ported to, or conceived in JavaScript, which is congruent with [Atwood's Law](http://blog.codinghorror.com/the-principle-of-least-power/). JavaScript does not only run in the browser, but you can also run JavaScript as a server side application or make it do a ton of crazy stuff with [node.js](https://nodejs.org/en/).
+
+## Mozilla articles
+Mozilla is the foundation behind the Firefox browser, they have an excellent set of documents on a big range of topics. Let's highlight a few:
+* [Introduction to HTML](https://developer.mozilla.org/en-US/Learn/HTML/Introduction_to_HTML)
+* [Introduction to CSS](https://developer.mozilla.org/en-US/Learn/CSS/Introduction_to_CSS)
+* [Introductory JavaScript](https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/JavaScript_basics)
+* [Intermediate JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript)
+* [Advanced JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Inheritance_and_the_prototype_chain)
+* The JavaScript Guide for programmers: [MDN JavaScript Guide](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/)
+
+## Frontend Development
+We dipped our toes in JavaScript a bit, and we've played with it in the browser. Now what? Well, what kind of software are you writing. In the first section I said web applications can be made with [hapi.js](https://hapijs.com). While this is true that is not the case here at Nelen & Schuurmans. Most of the web application's backend is written in Django. The frontend is a whole different ball game. And something you can't really do properly while using Python. (Yes, I said it.)
```

Comment:
Scheelt per site. Sommige frontends kosten te veel werk om ze in javascript te maken. Sommige frontends zijn véél beter als server-gegenereerde html met een javascript schil.

(Ja, ik hapte :-) )